test/avocado/selenium-machines-disks.py

```diff
@@ -8,6 +11,29 @@ class MachinesDisksTestSuite(MachinesLib):
     :avocado: tags=machines
     """
 
+    def prepare_disk(self):
```

Comment:
I think prepare_disk could be more generic, by talking as argument a pool name, and creating disks on that pools. Then you could call it twice to create disks on the other pools. This makes the function easier expendable.
However I don't veto this change, this is your code, and it's going to be maintained by you, so feel free to leave it as you wish.

Another idea is to start adding docstring in the helper function, so that we can remember what they are doing. (https://www.python.org/dev/peps/pep-0257/)
This is also just a suggestion.

Reply:
OK, if possible, i will also add it in 'todo' list, and commit it in next pull request.

Reply:
Sounds good!